pkg/cache/context_test.go

```diff
@@ -50,15 +50,53 @@ import (
 )
 
 const (
-	Host1  = "HOST1"
-	appID1 = "app00001"
-	appID2 = "app00002"
-	appID3 = "app00003"
-
-	pod1UID      = "task00001"
-	taskUID1     = "task00001"
-	pod1Name     = "my-pod-1"
+	Host1 = "host0001"
+	Host2 = "host0002"
+
+	appID1             = "app00001"
+	appID2             = "app00002"
+	appID3             = "app00003"
+	appID4             = "app00004"
+	appID5             = "app00005"
+	non_existing_appID = "app-none-exist"
+
+	uid1  = "uid_0001"
+	uid2  = "uid_0002"
+	uid3  = "uid_0003"
+	uid4  = "uid_0004"
+	uid5  = "uid_0005"
+	uid0n = "uid_0000"
```

Comment:
'uid0n' seems weird to me.   It's a typo?

Reply:
Many thanks. Update the typo in the new commit.